spec/models/list_item_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe ListItem do
+  describe "#display_title" do
+    it "shows the rummager title by default" do
+      stub_request(:get, RummagerHelper::SEARCH_ENDPOINT)
```

Comment:
It seems to have been something invented here, it's part of gds-api-adapters, you can find them here https://github.com/alphagov/gds-api-adapters/tree/master/lib/gds_api/test_helpers